merge

HD-Digital opened the pull request.

betacentauri commented:
Please merge this pull request instead of this one "ARM - create rootfs.tar.bz2 and kernel1.bin for use ofgwrite util"  https://github.com/HD-Digital/meta-gfutures/pull/2

HD-Digital committed:
hd60:
-add usb-dvb vtuner support
-fastboot: add 7segment display support
-7segment display add a~z (not all symbols can been project good on the display, just use it for some basic info like strt or rec)
-optimize audio video mediaplayer
-powerled on/off (/proc/stb/power/powerled)
-hisilicon does not support rc4/5 remote which causes to crash driver: echo 4 > /proc/stb/ir/rc/type
-fix driver crash when activate second transcoding.
-add patch: build fails when activate CONFIG_RC_DECODERS=y

HD-Digital committed:
hisil:
-resize image to 850mb
-update bootargs

betacentauri committed:
Create 2. image zip file which can be used by ofgwrite

HD-Digital committed:
hd60 use subdirectory with rootfs to support multiboot image.
update bootargs drivers and fastboot

HD-Digital committed:
hd60: add multiboot startup support

HD-Digital committed:
hd60: remove temporary startup_once untill new recovery/multiboot solution is available

HD-Digital committed:
Merge pull request #2 from betacentauri/master

Create 2. image zip file which can be used by ofgwrite
